select_command/_activity1.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Learn SELECT Command"
+description: "Learn SELECT command"
+date: 2020-07-17
+prereq: "none"
+difficulty: "Beginner"
+icon: ""
+draft: false
+translationKey: "sql_activity_1"
+---
+<!-- Links for javascript and CSS needed for drop down logic -->
+<link rel="stylesheet" href="../_activity1.css" type="text/css">
+</link>
+<script type="text/javascript" src="../_activity1.js">
+</script>
+
+# Mission 1: Save the Planet of Fun From Boredom!
+<font size="3"> Your first mission is to go to the Planet of Fun and save them from boredom! 

```

Comment:
Same thing here - please don't use font size.

Reply:
I'll make the changes then update 👍